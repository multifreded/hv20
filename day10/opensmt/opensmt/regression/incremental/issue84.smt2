(set-logic QF_LIA)
(declare-fun _substvar_1517_ () Bool)
(declare-fun _substvar_1296_ () Bool)
(declare-fun _substvar_1298_ () Bool)
(declare-fun _substvar_1609_ () Bool)
(declare-fun _substvar_1518_ () Bool)
(declare-fun _substvar_1300_ () Bool)
(declare-fun v0 () Bool)
(declare-fun v1 () Bool)
(declare-fun v2 () Bool)
(declare-fun v3 () Bool)
(declare-fun v4 () Bool)
(declare-fun v5 () Bool)
(declare-fun v6 () Bool)
(declare-fun v7 () Bool)
(declare-fun v8 () Bool)
(declare-fun v9 () Bool)
(declare-fun v10 () Bool)
(declare-fun v11 () Bool)
(declare-fun i2 () Int)
(declare-fun i3 () Int)
(declare-fun i5 () Int)
(declare-fun i7 () Int)
(declare-fun i10 () Int)
(declare-fun i13 () Int)
(declare-fun v12 () Bool)
(assert v4)
(check-sat)
(declare-fun v13 () Bool)
(assert (=> v13 v3))
(assert (=> v7 v2))
(declare-fun v14 () Bool)
(declare-fun i14 () Int)
(assert (distinct i7 i2))
(declare-fun v15 () Bool)
(declare-fun v16 () Bool)
(assert (xor (= i7 i14) v0))
(assert (= i7 i14))
(declare-fun v17 () Bool)
(assert (or (distinct i2 23) v8))
(declare-fun i15 () Int)
(push 1)
(assert (=> (distinct i7 i2) (xor true v0)))
(assert v1)
(declare-fun v19 () Bool)
(push 1)
(assert (= (= 98 i5) (distinct i2 23)))
(assert (=> v11 v2))
(push 1)
(declare-fun v22 () Bool)
(assert (=> (=> v13 v3) (distinct v11 (=> v11 v14))))
(push 1)
(push 1)
(check-sat)
(assert _substvar_1609_)
(assert v16)
(assert v9)
(assert (>= 176 (/ i13 858)))
(pop 1)
(assert v14)
(check-sat)
(declare-fun v28 () Bool)
(check-sat)
(assert (not v15))
(push 1)
(push 1)
(declare-fun v29 () Bool)
(assert (= false (not v12)))
(assert (not v28))
(push 1)
(assert (and (distinct v10 v4) v0))
(assert v29)
(pop 1)
(assert (xor v19 (distinct v10 v4)))
(check-sat)
(assert (not v22))
(check-sat)
(assert (or v13 (not v12)))
(check-sat)
(assert (=> v5 v1))
(assert (=> (= i3 (- i10)) (distinct v28 v8)))
(pop 1)
(pop 1)
(pop 1)
(assert (and v13 v17))
(pop 1)
(assert (= (<= i5 i13) v12))
(assert (> 23 i5))
(assert (= _substvar_1296_ v12))
(assert _substvar_1298_)
(assert _substvar_1300_)
(push 1)
(assert (= v7 v10))
(pop 1)
(pop 1)
(assert v7)
(assert (=> (=> (distinct i7 i2) (= i7 i14)) v19))
(assert (= (distinct i7 i2) v14))
(check-sat)
(assert (distinct v10 v19))
(pop 1)
(push 1)
(assert (= v10 v15))
(push 1)
(assert (distinct v14 (= 98 i5)))
(assert (or (= 98 i5) v7))
(declare-fun v59 () Bool)
(assert v59)
(push 1)
(declare-fun v60 () Bool)
(assert v60)
(declare-fun v61 () Bool)
(assert (or v3 (= i7 i14)))
(assert (= (= 98 i5) v61))
(push 1)
(check-sat)
(assert (and (distinct i7 759) (and v59 v4)))
(assert v16)
(check-sat)
(assert (< i5 59))
(pop 1)
(pop 1)
(pop 1)
(pop 1)
(declare-fun i29 () Int)
(declare-fun v74 () Bool)
(assert (and v8 v1))
(declare-fun v75 () Bool)
(assert (= 65 i3))
(push 1)
(push 1)
(assert (xor v3 (or v10 v74)))
(assert (or (= 65 i3) v10))
(check-sat)
(assert v17)
(push 1)
(declare-fun v79 () Bool)
(assert (xor v3 v13))
(assert (or v17 v12))
(assert (not v79))
(pop 1)
(assert (distinct (distinct (or (distinct i2 23) v8) v74) v6))
(declare-fun v82 () Bool)
(assert (= v82 v8))
(push 1)
(assert (or v15 v0))
(assert (or v8 (distinct i13 (- 65 i15))))
(assert (= (distinct v16 v10) v6))
(pop 1)
(assert (and (distinct v6 v8) (distinct v10 v4)))
(assert (or v10 v74))
(check-sat)
(check-sat)
(pop 1)
(assert (or v8 v0))
(check-sat)
(pop 1)
(declare-fun v91 () Bool)
(assert (distinct v16 (distinct i2 23)))
(check-sat)
(declare-fun v92 () Bool)
(assert (= v2 v75))
(assert (or (< i29 237) (>= i13 52)))
(declare-fun v95 () Bool)
(assert (xor v95 (and v8 v1)))
(declare-fun v96 () Bool)
(assert (= v11 (=> v13 v3)))
(assert v13)
(check-sat)
(declare-fun v100 () Bool)
(assert v2)
(assert (=> v12 (distinct (xor (= i7 i14) v0) v12)))
(assert (= (< (- (+ i13 i29)) i2) v11))
(assert (= v75 (or v10 v74)))
(push 1)
(assert false)
(check-sat)
(pop 1)
(assert (=> (=> v7 v2) v1))
(declare-fun v112 () Bool)
(push 1)
(assert (or (not (xor v16 v7)) (<= i5 i15)))
(pop 1)
(assert (and (distinct v16 (distinct i2 23)) v15))
(assert v112)
(assert (= (= v2 v75) v96))
(push 1)
(declare-fun v121 () Bool)
(assert v91)
(assert _substvar_1517_)
(assert v100)
(check-sat)
(assert (xor v121 _substvar_1518_))
(assert false)
(push 1)
(assert (=> v92 v6))
(check-sat)
(assert (=> (>= i14 52) (>= i13 34)))
(pop 1)
(check-sat)
(pop 1)
(assert true)
(push 1)
(check-sat)
(check-sat)

