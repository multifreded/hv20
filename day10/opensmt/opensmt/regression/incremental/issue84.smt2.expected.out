sat
sat
sat
sat
sat
sat
sat
sat
sat
sat
sat
unsat
unsat
sat
sat
sat
unsat
sat
unsat
unsat
sat
sat
